app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -252,6 +254,13 @@ def pod_full_name
     "#{options[:pod_namespace]}/#{options[:pod_name]}"
   end
 
+  def inspector_admin_secret
+    kubeclient = kubernetes_client
+    sas = kubeclient.get_service_accounts
```

Comment:
@enoodle I think you can use `kubeclient.get_service_account(IMAGE_INSPECTOR_SA, INSPECTOR_NAMESPACE)` or something similar.

Also plan for the SA to be missing and log the fact that you couldn't collect the `imagePullSecrets` (but try to continue anyway without the volume).